at-chuongphu/excercise5.rb

```diff
@@ -0,0 +1,51 @@
+# ex1
+class Student
+  attr_accessor :first_name, :last_name, :list_course
+  def initialize
+    @list_course = []
+  end
+
+  def first_name
+    puts @first_name.to_s
+  end
+
+  def last_name
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/834451464516aa55ed2fa56b4496badd74fc940e/at-chuongphu/excercise5.rb#L12"></span>Lint/DuplicateMethods: Method <code>Student#last_name</code> is defined at both at-chuongphu/excercise5.rb:3 and at-chuongphu/excercise5.rb:12.</td>
    </tr>
  </tbody>
</table>